src/pg-storage-client.ts

```diff
@@ -185,6 +186,23 @@ export default class PGStorageClient {
     async writeBotCounter(userId: Snowflake, counter: number): Promise<void> {
         await this.client.query('INSERT INTO bot_counters VALUES ($1, $2) ON CONFLICT (user_id) DO UPDATE SET counter = EXCLUDED.counter;', [userId, counter]);
     }
+
+    async fetchAllPrivilegedRoles(): Promise<Record<Snowflake, Snowflake>> {
+        const result: Record<Snowflake, Snowflake> = {};
+        const queryResult = await this.client.query<{guild_id: Snowflake, role_id: Snowflake}>('SELECT * FROM privileged_roles;');
+        for (const row of queryResult.rows) {
+            result[row.guild_id] = row.role_id;
+        }
+        return result;
+    }
+    
+    async deletePrivilegedRole(guildId: Snowflake, roleId: Snowflake): Promise<void> {
```

Comment:
Looks like this isn't used now. Are we going to use this when being removed from guilds? Perhaps later we can implement some `deleteAllGuildData` method